src/ploomber/sources/notebooksource.py

```diff
@@ -393,34 +392,31 @@ def _post_init_validation(self, value):
         Validate notebook after initialization (run pyflakes to detect
         syntax errors)
         """
-        # NOTE: what happens if I pass source code with errors to parso?
-        # maybe we don't need to use pyflakes after all
-        # we can also use compile. can pyflakes detect things that
-        # compile cannot?
+        pass
+
+    def _validate_parameters_cell(self,
+                                  extract_upstream=False,
+                                  extract_product=False):
+        '''Check parameters call and add it when it's missing
+
+        Keyword arguments:
+        extract_upstream -- Flags used to determine the content of
+        the parameters cell, only used if the notebook is
+        missing the parameters cell (default False)
+        extract_product -- Same as extract_upstream (default False)
+        '''
```

Comment:
please use numpydoc format for the docstring https://numpydoc.readthedocs.io/en/latest/format.html

Reply:
Updated:
        '''Check parameters call and add it when it's missing

        Parameters
        ----------
        extract_upstream : bool, default: True
            Flags used to determine the content of the parameters cell,
            only used if the notebook is missing the parameters cell
        extract_product : bool, default: True
            Same as extract_upstream (default False)
        '''